### PR TITLE
Issue 1 videosearch #1 Completed

### PR DESCRIPTION
**Pull request by swedendrift**
Users can currently search a static array of objects for a single search term.  If a match is found in the title or description the object is rendered onto the page.  items are currently not linked to actual videos and only display a thumbnail and text.

### CODE REVIEW

File: css/default.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  margin: 0;
+  width: 40%;
+  font-size: 18px;
+  appearance: none;
+  box-shadow: none;
+  border-radius: none;
+}
+
+input[type="button"] {
+  	border : solid 1px #e0e0e0;
```
Comment (thebearingedge) [inline css/default.css]: What happened with your indentation here?

File: css/default.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  	border-radius : 0px 0px 1px 0px ;
+  	moz-border-radius : 0px 0px 1px 0px ;
+  	font-size : 18px;
+  	color : #000000;
+  	padding : 2px 6px;
+    background-color: inherit;
+}
+
+#queryEcho {
```
Comment (thebearingedge) [inline css/default.css]: We use `kebab-case` for CSS class names.

File: css/default.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  float: left;
+  padding-top: 20px;
+}
+
+#container ul {
```
Comment (thebearingedge) [inline css/default.css]: See if you can give these elements more meaningful names, e.g.
`results-list`
`thumbnail`
`results-item`

File: js/default.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
+//create an array of objects to represent the videos (pictures right atm)
+var videos = [{title: "surf", videographer: "Mowgly Lee", description: 'Nulla facilisi. Sed vel erat ut augue surf surfing vel eu nibh. Pellentesque pellentesque dolor felis, at iaculis sem blandit non. Aenean ut ipsum id ex laoreet finibus a ut.', url: "imgs/surf.jpg"},
```
Comment (thebearingedge) [inline js/default.js]: Use some new lines and indentation to make your data easier to read.

File: js/default.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+{title: "nascar", videographer: "Robert Denton", description: 'Nulla facilisi. Sed vel car ut augue nascar elementum vel eu crash. Pellentesque pellentesque dolor felis, at iaculis sem blandit non. Aenean ut ipsum id ex laoreet finibus a ut.', url: "imgs/nascar.jpg"},
+{title: "volleyball", videographer: "Bruce Chindo", description: 'Nulla volleyball vel erat ut augue volley ball elementum vel eu nibh. Pellentesque pellentesque dolor felis, at iaculis sem blandit non. Aenean ut ipsum id ex laoreet finibus a ut.', url: "imgs/volleyball.jpg"},
+{title: "surf", videographer: "Chris Clark", description: 'Nulla facilisi. Sed vel surf ut augue sollicitudin elementum vel surfing. Pellentesque drunk dolor felis, at iaculis sem blandit non. Aenean ut ipsum id ex laoreet finibus a ut.', url: "imgs/surf2.jpg"},
+{title: "nascar", videographer: "Robert Denton", description: 'Nulla facilisi. Sed vel car ut augue nascar elementum vel eu crash. Pellentesque pellentesque dolor felis, at iaculis sem surf non. Aenean ut ipsum id ex laoreet finibus a ut.', url: "imgs/nascar.jpg"},];
+
+
+// function to add new video results to the page
```
Comment (thebearingedge) [inline js/default.js]: Remove comments and give your functions meaningful names. e.g., `appendVideo`

File: css/default.css (hunk 2 above)
Comment (swedendrift) [inline css/default.css]: changed to query-echo

File: css/default.css (hunk 3 above)
Comment (swedendrift) [inline css/default.css]: changed to container to results-container

File: css/default.css (hunk 1 above)
Comment (swedendrift) [inline css/default.css]: fixed.

File: js/default.js (hunk 5 above)
Comment (swedendrift) [inline js/default.js]: functions are more descriptive (addNewItem is now addSearchResult)

**Pull request merged**